### ISSUE TITLE
Call to a member function dropExperience() on null

### ISSUE THREAD

**Issue by LousWiteMC** [RevivalPMMP/PureEntitiesX#382]
[Server thread/CRITICAL]: Error: "Call to a member function dropExperience() on null" (EXCEPTION) in "src/pocketmine/entity/Living" at line 623

**Comment by 95CivicSi** [comment]
I can't begin to resolve issues like this unless you fill out the information that's requested in the issue Template.  Please see https://github.com/RevivalPMMP/PureEntitiesX/wiki/Issue-Reporting-Guidelines

Feel free to re-open this when the original post has been updated, or open a new issue with the template properly filled out.
